### ISSUE TITLE
Search bar overlaps the sign-in field and is missing a close button when logged out

### ISSUE THREAD

**Issue by wisherp** [CenterForOpenScience/osf.io#4159]
# Steps
1. Go to osf.io
2. Log out if already logged in
3. Click on search 
4. Search for anything, for example 'Science'
5. Now click on Sign In button at the top 

# Expected
The sign in field appears on top of the search bar so that the user could log in. Also the search bar should have a 'x' next to the '?' to close it.

# Actual
 The search bar is missing an 'x' and also overlaps the sign in field which prevents the user from logging in.

![search bar overlaps the sign-in field](https://cloud.githubusercontent.com/assets/10926926/9450819/cea2e914-4a78-11e5-8786-237b8f50e240.png)

**Comment by haoyuchen1992** [comment]
I can look at it, Thanks!

**Comment by lbanner** [comment]
You got it!

**Comment by caneruguz** [comment]
Thanks to @haoyuchen1992 who came to talk to me about this before we went further with this issue. 

The overlapping issue is already open in this one: https://github.com/CenterForOpenScience/osf.io/issues/4088
I think it's a complete duplicate since 4088 covers the real more general problem that the sign in drowdown is part of. 

The second issue of the X is intentional. We did it this way with @brianjgeiger because this is the search page and search bar is a permanent part of this page. We think this provides better continuity across pages. I think if users are confused about this we should change it but that should be lead by actual user information. 

So this issue should be closed since it is a duplicate and includes something that can't be acted on. Also for future we should avoid including multiple problems into the same issue. They may be quite unrelated as in this case. 

@lbanner
